static/js/common/fxa-login.js

```diff
@@ -0,0 +1,62 @@
+(function() {
+    function getClient() {
+        return new FxaRelierClient(config.clientId, {
+            contentHost: config.contentHost,
+            oauthHost: config.oauthHost,
+        });
+    }
+
+    var fxaConfig = {
+        "http://olympia.dev": {
+            "clientId": "cd5a21fafacc4744",
+            "contentHost": "https://stable.dev.lcip.org",
+            "loginUri": "http://olympia.dev/api/v3/login/",
+            "oauthHost": "https://oauth-stable.dev.lcip.org/v1",
+            "redirectUri": "http://olympia.dev/fxa-authorize",
+            "scope": "profile",
+        },
+        "http://localhost:8000": {
+            "clientId": "a1dc6167e57ad856",
+            "contentHost": "https://stable.dev.lcip.org",
+            "loginUri": "http://olympia.dev/api/v3/login/",
+            "oauthHost": "https://oauth-stable.dev.lcip.org/v1",
+            "redirectUri": "http://localhost:8000/fxa-authorize",
+            "scope": "profile",
+        },
+    };
+    var config = fxaConfig[location.origin];
+    if (!config) {
+        console.log("[FxA] No FxA config found for " + location.origin);
+    }
+    var fxaClient = getClient();
+
+    window.fxaLogin = function() {
```

Comment:
we have [AMD module support](https://github.com/mstriemer/olympia/blob/fxa-config/static/js/common/buckets.js#L1). Can you use that instead?